clap_mangen/src/render.rs

```diff
@@ -119,13 +123,18 @@ pub(crate) fn options(roff: &mut Roff, cmd: &clap::Command) {
     }
 
     for pos in items.iter().filter(|a| a.is_positional()) {
-        let (lhs, rhs) = option_markers(pos);
-        let name = format!("{}{}{}", lhs, pos.get_id(), rhs);
-
-        let mut header = vec![bold(&name)];
-
+        let mut header = vec![];
         let mut body = vec![];
```

Comment:
nit: do you mind moving `body` down to right before its used, effectively dividing the function into `header` and `body` parts?